oak/proto/application.proto

```diff
@@ -71,3 +72,11 @@ message StorageProxyConfiguration {
   // The address of the external storage provider.
   string address = 1;
 }
+
+// GrpcClientConfiguration describes the configuration of a gRPC client
+// pseudo-Node (which is provided by the Oak Runtime), connected to a specific
+// external (non-Oak) gRPC service.
+message GrpcClientConfiguration {
```

Comment:
I think we chatted briefly in person about this, but what are your thoughts about extending this definition, vs having something more expressive in the runtime, by which e.g. a node may call a function to request the creation of a gRPC client pseudo-node with a specific config? The config may still be expressed as a protobuf for now, but in the future it may be in a more expressive IDL.